setup/paths.sh

```diff
@@ -37,38 +37,25 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
+
+# Setup Path
+export PATH=$PATH:$GEM_PLOTTING_PROJECT
+
+# Setup PYTHONPATH
+export PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts
```

Comment:
As elsewhere, if this should be in the `$PYTHONPATH`, it should be set up that way in `vfatqc-python-scripts`